.dockerignore

```diff
@@ -0,0 +1 @@
+example
```

Comment:
this excludes `example` (similar to `.gitignore`) from the image build context. Basically `docker build` command won't see the folder. It is useful when you don't want to let something sensitive to leak or if the directory/file is real big and simply loading it into the build context will take a lot of time.